lib/tutorial/chapter_six.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:math';
+
+void main() {
+
+  print("Mini-exercises");
+
+  print("-----------------------------------------------------------------------");
+  print("1. Create a class called Password and give it a string property called value.");
+  print("2. Override the toString method of Password so that it prints value.");
+
+  final newPassword = Password();
+  newPassword.value = '218176';
+  print(newPassword);
+
+  print("3. Add a method to Password called isValid that returns true only if the length of value is greater than 8.");
+
+  int lenght = newPassword.value.length;
+  print('Value lenght is $lenght.');
+  print(newPassword.isValid());
+  print("-----------------------------------------------------------------------");
+
+  print(" Given \"newPassord\" class");
+  print("1. Make value a final variable, but not private.");
+  print("2. Add a const constructor as the only way to initialize newPassword object.");
+  print("-----------------------------------------------------------------------");
+
+  print("-----------------------------------------------------------------------");
+  print("Challenges");
+  print("-----------------------------------------------------------------------");
+
+  print("Challenge 1: Bert and Ernie");
+  print("1. Create a Student class with final firstName and lastName String properties and a variable grade as an int property.");
+  print("2. Add a constructor to the class that initializes all the properties. ");
+  print("3. Add a method to the class that nicely formats a Student for printing.");
+  print("4. Use the class to create students bert and ernie with grades of 95 and 85, respectively.");
+
+  final bert = Student("Bert", "No last name", 95);
```

Comment:
@octavianChiriches final Student bert, please always declare the types for variables.